### PR TITLE
Makes the "Player has a matching cookie" message that admins get more visible.

### PR DESCRIPTION
**Pull request by Fruktlimpa**
By request and by experience this PR was created to make the message more visible for us to more easily catch those ban evading.
This is just a quick-fix and there are most likely better ways to enhance this but this will have to do for now.

dreamseeker_2017-08-25_21-19-45

:cl:
 * rscadd: Made the admin "Cookie message" more visible

### CODE REVIEW

File: goon/code/datums/browserOutput.dm [hunk 1]
```diff
@@ -175,7 +175,7 @@ For the main html chat area
 			//Uh oh this fucker has a history of playing on a banned account!!
 			if (found.len > 0)
 				//TODO: add a new evasion ban for the CURRENT client details, using the matched row details
-				message_admins("[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])")
+				message_admins("<FONT SIZE=+2 COLOR=red>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
```
Comment (jknpj) [inline goon/code/datums/browserOutput.dm]: Might as well make it send this alert to discord. Also please avoid using COLOR=red, there's already the danger class for red+bold.

Comment (Exxion) [reply goon/code/datums/browserOutput.dm]: I'd suggest not sending it to Discord until its tendency to report that a person is themselves is fixed

Comment (Fruktlimpa) [reply goon/code/datums/browserOutput.dm]: We get a lot of false positives so I'm not sure a discord send would be good at this point but should that ever be fixed I could definitely work on getting it to send to discord.
As for the color, what do you suggest I change? Just use the hex code for red or some other color?

Comment (Exxion) [reply goon/code/datums/browserOutput.dm]: `<span class='danger'>`

File: goon/code/datums/browserOutput.dm [hunk 2]
```diff
@@ -175,7 +175,7 @@ For the main html chat area
 			//Uh oh this fucker has a history of playing on a banned account!!
 			if (found.len > 0)
 				//TODO: add a new evasion ban for the CURRENT client details, using the matched row details
-				message_admins("<FONT SIZE=+2 COLOR=red>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
+				message_admins("<FONT SIZE=+2 span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
```
Comment (Intigracy) [inline goon/code/datums/browserOutput.dm]: Still incorrect.
Now: `message_admins("<FONT SIZE=+2 span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")`

Should be: `message_admins("<span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</span>")`

Does it even work the way it is right now?

**Review by Intigracy — DISMISSED** [review]

File: goon/code/datums/browserOutput.dm (hunk 2 above)
Comment (Fruktlimpa) [reply goon/code/datums/browserOutput.dm]: I did test it before sending the commit and it did properly display it. But I'll change it up again tomorrow.

Comment (Fruktlimpa) [reply goon/code/datums/browserOutput.dm]: This is the result after changing it and it isn't as large as I wanted it to be. The reason I used font size was to make it larger than the regular text.

dreamseeker_2017-08-27_17-39-57

**Comment by Fruktlimpa** [comment]
Going to fix a thing later today so do not merge just yet.

**Comment by Fruktlimpa** [comment]
The thing compiled regardless of the < /span > and worked regardless. Adding it just in case. Ready to merge from my end, tested and all.

File: goon/code/datums/browserOutput.dm [hunk 3]
```diff
@@ -175,7 +175,7 @@ For the main html chat area
 			//Uh oh this fucker has a history of playing on a banned account!!
 			if (found.len > 0)
 				//TODO: add a new evasion ban for the CURRENT client details, using the matched row details
-				message_admins("[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])")
+				message_admins("<FONT SIZE=+2 span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</span></FONT>")
```
Comment (N3X15) [inline goon/code/datums/browserOutput.dm]: get rid of the FONT SIZE=+2 shit and the `</FONT>`
`<FONT>` tags are outdated.  `<span>` tags use CSS, which is more flexible.
Should look like `<span class='danger'>Blah</span>`

**Review by N3X15 — CHANGES_REQUESTED** [review]

File: goon/code/datums/browserOutput.dm (hunk 3 above)
Comment (Fruktlimpa) [reply goon/code/datums/browserOutput.dm]: I already tried span class danger and it wasn't as large as I wanted it to be. See the screenshot above. 
Simply span class danger doesn't increase the size which is why I used font size.

Comment (DamianX) [reply goon/code/datums/browserOutput.dm]: There's a `big` class defined in the spreadsheet, you could try something like `<span class='danger big'>`, instead.

Comment (Fruktlimpa) [reply goon/code/datums/browserOutput.dm]: Going to try that then and we'll see. I just hope it isn't Nar-sie has risen big.

**Comment by Fruktlimpa** [comment]
Seems like 'danger big' did the trick. It looks like font size+2. But better.
poo

**Review by N3X15 — APPROVED** [review]

**Pull request merged**